### PR TITLE
[PATCH] Fix CAPA Problems

### PR DESCRIPTION
**Pull request by OmarIthawi**
As received via email with the same title from edX.

### CODE REVIEW

**Comment by melvinsoft** [comment]
Thanks @OmarIthawi !

**Comment by johnbaldwin** [comment]
@OmarIthawi What does CAPA stand for? I understand they are problems/exercises. I web searched and see it referenced in some places in edX pages, but none I look at actually spells out the acronym

**Comment by OmarIthawi** [comment]
> What does CAPA stand for? I understand they are problems/exercises. I web searched and see it referenced in some places in edX pages, but none I look at actually spells out the acronym

Great question @johnbaldwin! It makes a nice brotherhood name.

I don't know, so I had to look it up. It looks like it's a short for [LON-CAPA](https://en.wikipedia.org/wiki/LON-CAPA), which is a short for: "Learning Online Network with Computer-Assisted Personalized Approach". A circa 2007 LMS.

In the [Open edX context](https://open.edx.org/course-staff/upcoming-markdown-and-olx-changes-capa-problems/) it means: "(multiple choice, drop down, etc)".

https://github.com/appsembler/edx-platform/blob/facccb96a70fdf879b0cddb74880c7d1df8ed3f0/common/lib/xmodule/xmodule/capa_module.py#L22-L29

So basically edX copied whatever there's in LON-CAPA in terms of [both problem types and underlying format](https://github.com/edx/edx-documentation/blob/master/en_us/olx/source/components/problem-components.rst#problem-component-overview).

<kbd>image</kbd>
